content/200-orm/200-prisma-client/100-queries/090-raw-database-access/050-raw-queries.mdx

```diff
@@ -550,20 +524,184 @@ The database will thus provide a `String` representation of your data which Pris
 
 For details of supported Prisma types, see the [Prisma connector overview](/orm/overview/databases) for the relevant database.
 
-### SQL injection
+## SQL injection
```

Comment:
Is there a way to describe this in a more positive way? `Avoiding SQL injections` comes to mind, but maybe there is something even more positive?

Reply:
I tried `SQL injection prevention`